decred/decred/dcr/rpc.py

```diff
@@ -943,6 +1171,120 @@ def get(k, obj):
     return obj[k] if k in obj else None
 
 
+class COut:
```

Comment:
A little description would be good here, especially since the class name is not very descriptive.